go.mod

```diff
@@ -1,11 +1,13 @@
 module github.com/lightningnetwork/lnd
 
+replace github.com/btcsuite/btcwallet v0.16.1 => github.com/mstreet3/btcwallet v0.0.0-20221202121645-53904c1b1052
```

Comment:
replace to [here](https://github.com/btcsuite/btcwallet/pull/822/commits/53904c1b1052097ad5ea97364ee7288786cc7b71)